lib/modules/userTagger.js

```diff
@@ -381,6 +381,8 @@ export class Tag {
 			// `defaultTagElement` is a lot fast than `buildTagElement`
 			instance.tagger = (this.text || this.color) ? Tag.buildTagElement(this) : Tag.defaultTagElement();
 			instance.tagger.addEventListener('click', () => this.openPrompt(instance));
+			// Remove the previously added tags
+			instance.element.innerHTML = '';
```

Comment:
Instead of "blanking out" the element, it would be better to remove the element entirely. e.g. `instance.element.parentNode.removeElement(instance.element)`

Reply:
That said, this doesn't feel quite right .. I assumed that a _previous_ instance's element is sticking around, and that's what needs to be removed?

Reply:
Yeah, you are right. The previous instance's element has a tag that is not removed when a new post is loaded.

Reply:
@andytuba Removing the element won't work because every instance refers to the same container (`instance.element`) to insert the tag. I tried removing the element, and the tags did not show up. Please advise.

Reply:
Sorry, I should have said "some other instance's `.tagger`" element. 

`instance.element` is the JSAPI container, `<span data-name="reddit-enhancement-suite">...</span>`.  All RES's modules share that `instance.element`. So, UserTagger shouldn't clear out the entire `instance.element` because that might clobber content added by other RES modules.  Instead, UserTagger should only remove `.tagger` elements from this `instance.element`

Reply:
Turns out all instances are actually appending children in the same `<span data-name="reddit-enhancement-suite">...</span>`, because every instance's element points to it.

*User tags repeated inside the span*
![Screenshot from 2019-03-13 22-39-13](https://user-images.githubusercontent.com/20063271/54300028-12b09a80-45e2-11e9-81cf-6ed9d3319243.png)

Reply:
Exactly :)

Reply:
So, what I'm hoping you can help with, is to removing the old `.RESUserTag` (`instance.tagger`) when adding a new `tagger`.

Reply:
Yeah, so I need to clear the `instance.element` before appending more tags.

`if (instance.tagger) instance.tagger.remove();` : This check already exists

Do keep in mind that the tags being repeated are of different users and not the same user.

Reply:
... clear `.RESUserTag` elements from `instance.element` before appending more tags.  (Don't clear it out entirely.)

Yeah, that `if instance.tagger` check is checking for "same user".  The extra `.RESUserTag` items are from different users -- that's why they should be removed.

Reply:
Okay, got it.